src/test/codegen/union-abi.rs

```diff
@@ -63,11 +63,16 @@ pub union UnionU128{a:u128}
 #[no_mangle]
 pub fn test_UnionU128(_: UnionU128) -> UnionU128 { loop {} }
 
+pub union UnionU128x2{a:(u128, u128)}
+// CHECK: define void @test_UnionU128x2(i128 %_1.0, i128 %_1.1)
+#[no_mangle]
+pub fn test_UnionU128x2(_: UnionU128x2) { loop {} }
+
 #[repr(C)]
-pub union CUnionU128{a:u128}
-// CHECK: define void @test_CUnionU128(%CUnionU128* {{.*}} %_1)
+pub union CUnionU128x2{a:(u128, u128)}
+// CHECK: define void @test_CUnionU128x2(%CUnionU128x2* {{.*}} %_1)
 #[no_mangle]
-pub fn test_CUnionU128(_: CUnionU128) { loop {} }
+pub fn test_CUnionU128x2(_: CUnionU128x2) { loop {} }
```

Comment:
This was testing that single-field `#[repr(C)]` unions don't forward that field's ABI, but after this change `CUnionU128` would get passed by value, making it impossible to distinguish between `i128` [forwarded from inner field] vs `i128` [passing entire union by value]. So I had to make it bigger.